app/views/comments/_single_comment.html.erb

```diff
@@ -54,12 +56,12 @@
           <p class="notice"><%= ts("This comment has been hidden by an admin.") %></p>
         <% end %>
         <blockquote class="userstuff"><%=raw sanitize_field(single_comment, :comment_content) %></blockquote>
-        <% unless single_comment.edited_at.blank? %>
-          <p class="edited datetime">
-            <%= ts("Last Edited") %> <%= time_in_zone(single_comment.edited_at) %>
-          </p>
-        <% end %>
-      <% end %><!-- end caching -->
+      <% end %>
+      <% unless single_comment.edited_at.blank? %>
```

Comment:
We could improve the expression to be more direct here `if single_comment.edited_at.present?`.